pages/components/input/input.css

```diff
@@ -0,0 +1,50 @@
+@import "../../../styles/typography.css";
+@import "../../../styles/variable.css";
+@import "../../../styles/reset.css";
+@import "../../../styles/header.css";
+@import "../../../styles/drawer.css";
+@import "../../../styles/padding.css";
+@import "../../../styles/margin.css";
+@import "../../../styles/utils.css";
+
+.input-wrapper {
+  width: 30vw;
+  border: 1px solid lightgray;
+  border-radius: 5px;
+}
+
+.text-input {
+  width: 20vw;
+  border: 2px solid lightgray;
```

Comment:
Use variable for the color.

Reply:
Only for that color, I am not using variables because this color is not repeating much.